lib/panda/item.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Panda
+  class Item
+    def initialize(response)
+      response.parsed_body.fetch('data', {})
```

Comment:
Pretty sure we won't have access to this data that way

Reply:
Let's add test that we have access to the result of token check call

Reply:
Done